src/WorkOS.net/Services/UserManagement/Entities/MagicAuthChallenge.cs

```diff
@@ -0,0 +1,23 @@
+namespace WorkOS
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Represents a request for a Magic Auth code.
+    /// </summary>
+    public class MagicAuthChallenge
```

Comment:
Is this being used anywhere? It doesn't seem to be referenced.

Reply:
Forgot to remove delete this file, it's references were removed, deleting file.